specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -556,12 +556,63 @@
         },
         "properties": {
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/StorageTaskProperties",
+          "$ref": "#/definitions/StorageTaskUpdateProperties",
           "description": "Properties of the storage task."
         }
       },
       "description": "Parameters of the storage task update request"
     },
+    "StorageTaskUpdateProperties": {
+      "type": "object",
+      "properties": {
+        "taskVersion": {
+          "readOnly": true,
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 1,
+          "format": "int64",
```

Comment:
consider using an enum to allow for future extensibility of values beyond 2 states